packages/core/src/docdb/commands/renameCluster.ts

```diff
@@ -54,6 +55,7 @@ export async function renameCluster(node: DBClusterNode) {
                 localize('AWS.docdb.renameCluster.success', 'Updated cluster: {0}', clusterName)
             )
 
+            await sleep(1000)
```

Comment:
Let's add a comment to give this some context for any future devs that stumble across it. As it is now it seems arbitrary.

Reply:
done